src/video/wayland/SDL_waylandevents.c

```diff
@@ -64,7 +64,7 @@
 
 /* Clamp the wl_seat version on older versions of libwayland. */
 #if SDL_WAYLAND_CHECK_VERSION(1, 21, 0)
-#define SDL_WL_SEAT_VERSION 8
+#define SDL_WL_SEAT_VERSION 9
 #else
 #define SDL_WL_SEAT_VERSION 5
 #endif
```

Comment:
This might need to be expanded with another version check for whatever version of libwayland has seat ver 9, as compositors can expose higher protocol versions to applications than older versions of libwayland can handle, which can cause problems.

See issue #6516 that originally necessitated this check.